fedora/python-ogr.spec

```
@@ -52,6 +52,9 @@ rm -rf %{srcname}.egg-info
 
 
 %changelog
+* Thu Jul 09 2020 Jiri Popelka <jpopelka@redhat.com> - 0.12.2-1
+- 0.12.2
```

Maybe change this, for consistency.

```suggestion
- New upstream release 0.12.2
```